src/extensions/scratch3_ev3/index.js

```diff
@@ -943,7 +968,11 @@ class Scratch3Ev3Blocks {
                 },
                 {
                     opcode: 'whenBrightnessLessThan',
-                    text: 'when brightness < [DISTANCE]',
+                    text: formatMessage({
+                        id: 'ev3.whenBrightnessLessThan',
+                        default: 'when brightness < [DISTANCE]',
```

Comment:
@ericrosenbaum is this the correct block command? with the placeholder `[DISTANCE]` when the block is brightness? (Localization part is fine 👍 )